app/data/session-data-defaults.js

```diff
@@ -16,9 +16,47 @@ Example usage:
 ============================================================================
 
 */
-
 module.exports = {
-
-  // Insert values here
-
+  // Example as used in current docs
```

Comment:
I think we need to leave this file blank for users to fill in if they need to? Can this test data be stored somewhere else?

Reply:
It's just test code for whoever picks this up next.

Reply:
ah thanks!